apps/frontend/src/pages/projects/components/project-card/project-card.tsx

```diff
@@ -16,9 +18,14 @@ const ProjectCard = ({ project }: Properties): JSX.Element => {
 	});
 
 	return (
-		<NavLink className={styles["project"] ?? ""} to={projectRoute}>
-			{project.name}
-		</NavLink>
+		<div className={styles["project"] ?? ""}>
+			<NavLink className={styles["project-link"] ?? ""} to={projectRoute}>
+				{project.name}
+			</NavLink>
+			<ProjectPopover project={project}>
+				<Icon height={20} name="options" width={20} />
```

Comment:
This should be `IconButton` instead of `Icon`

Reply:
Please take a look at these two discussions regarding IconButton and the options menu overall, it would be nice to have a separate component for options menu

https://bsa2024.slack.com/archives/C07HWM8616U/p1725289146857129
https://bsa2024.slack.com/archives/C07HWM8616U/p1725290415052879